ios/RNLinksdk.m

```diff
@@ -576,26 +577,196 @@ + (NSString *)subtypeNameForAccountSubtype:(id<PLKAccountSubtype>)accountSubtype
+        },
+        kPLKExitErrorApiDomain: @{
+          @(PLKApiErrorCodeInternalServerError): @"INTERNAL_SERVER_ERROR",
+          @(PLKApiErrorCodePlannedMaintenance): @"PLANNED_MAINTENANCE",
+        },
+        kPLKExitErrorItemDomain: @{
+          @(PLKItemErrorCodeInsufficientCredentials): @"INSUFFICIENT_CREDENTIALS",
+          @(PLKItemErrorCodeInvalidCredentials): @"INVALID_CREDENTIALS",
+          @(PLKItemErrorCodeInvalidMfa): @"INVALID_MFA",
+          @(PLKItemErrorCodeInvalidSendMethod): @"INVALID_SEND_METHOD",
+          @(PLKItemErrorCodeInvalidUpdatedUsername): @"INVALID_UPDATED_USERNAME",
+          @(PLKItemErrorCodeItemLocked): @"ITEM_LOCKED",
+          @(PLKItemErrorCodeItemLoginRequired): @"ITEM_LOGIN_REQUIRED",
+          @(PLKItemErrorCodeItemNoError): @"ITEM_NO_ERROR",
+          @(PLKItemErrorCodeItemNotSupported): @"ITEM_NOT_SUPPORTED",
+          @(PLKItemErrorCodeIncorrectDepositAmounts): @"INCORRECT_DEPOSIT_AMOUNTS",
+          @(PLKItemErrorCodeUserSetupRequired): @"USER_SETUP_REQUIRED",
+          @(PLKItemErrorCodeMfaNotSupported): @"MFA_NOT_SUPPORTED",
+          @(PLKItemErrorCodeNoAccounts): @"NO_ACCOUNTS",
+          @(PLKItemErrorCodeNoAuthAccounts): @"NO_AUTH_ACCOUNTS",
+          @(PLKItemErrorCodeNoInvestmentAccounts): @"NO_INVESTMENT_ACCOUNTS",
+          @(PLKItemErrorCodeNoLiabilityAccounts): @"NO_LIABILITY_ACCOUNTS",
+          @(PLKItemErrorCodeProductNotReady): @"PRODUCT_NOT_READY",
+          @(PLKItemErrorCodeProductsNotSupported): @"PRODUCTS_NOT_SUPPORTED",
+          @(PLKItemErrorCodeInstantMatchFailed): @"INSTANT_MATCH_FAILED",
+        },
+        kPLKExitErrorAuthDomain: @{
+          @(PLKAuthErrorCodeProductNotReady): @"PRODUCT_NOT_READY",
+          @(PLKAuthErrorCodeVerificationExpired): @"VERIFICATION_EXPIRED",
+        },
+        kPLKExitErrorAssetReportDomain: @{
+          @(PLKAssetReportErrorCodeProductNotEnabled): @"PRODUCT_NOT_ENABLED",
+          @(PLKAssetReportErrorCodeDataUnavailable): @"DATA_UNAVAILABLE",
+          @(PLKAssetReportErrorCodeProductNotReady): @"PRODUCT_NOT_READY",
+          @(PLKAssetReportErrorCodeAssetReportGenerationFailed): @"ASSET_REPORT_GENERATION_FAILED",
+          @(PLKAssetReportErrorCodeInvalidParent): @"INVALID_PARENT",
+          @(PLKAssetReportErrorCodeInsightsNotEnabled): @"INSIGHTS_NOT_ENABLED",
+          @(PLKAssetReportErrorCodeInsightsPreviouslyNotEnabled): @"INSIGHTS_PREVIOUSLY_NOT_ENABLED",
+        },
+        kPLKExitErrorInternalDomain: error.userInfo[kPLKExitErrorCodeKey] ?: @"UNKNOWN",
+        kPLKExitErrorUnknownDomain: error.userInfo[kPLKExitErrorCodeKey] ?: @"UNKNOWN",
+    };
+    return @"unknown";
+}
+
++ (NSString *)errorMessageFromError:(PLKExitError *)error {
+    return error.userInfo[kPLKExitErrorMessageKey] ?: @"";
+}
+
++ (NSString *)stringForEventName:(PLKEventName *)eventName {
+    if (!eventName) {
+        return @"";
+    }
+    
+    if (eventName.unknownStringValue) {
```

Comment:
`&& [eventName.unknownStringValue length]` ?
Since unknownStringValue can be empty?

Reply:
Hrm, this is a little tricky – if the iOS SDK returns .unknown("") we probably do want to have this result in @"" being returned here.